config/locales/en.yml

```diff
@@ -436,11 +436,11 @@ en:
           - "Yes"
           - "Not sure"
         items:
-          - text: "What to do if you have lost your job"
+          - text: "What to do if you've lost your job"
```

Comment:
```suggestion
          - text: "What to do if you‘ve lost your job"
```

Reply:
yep, should be the type of apostrophes we use, wasn't intentional!